README.md

```diff
@@ -9,7 +9,7 @@ The most recent development builds can be found [here](https://github.com/doitsu
 Release builds can be found [here](https://github.com/doitsujin/dxvk/releases).
 
 ## How to use
-In order to install a DXVK package obtained from the [release](https://github.com/doitsujin/dxvk/releases) page into a given wine prefix, copy or symlink the DLLs into the following directories as follows, then open `winecfg` and manually add DLL overrides for `d3d11`, `d3d10core`, `dxgi`, `d3d9` and `d3d8`.
+In order to install a DXVK package obtained from the [release](https://github.com/doitsujin/dxvk/releases) page into a given wine prefix, copy or symlink the DLLs into the following directories as follows, then open `winecfg` and manually add DLL overrides for `d3d11`, `d3d10core`, `dxgi`, `d3d9` and `d3d8`. To use d3d8 with dxvk you have to use both the d3d8 and d3d9 DLLs as d3d8.dll depends on d3d9.dll.
```

Comment:
I'm not sure there is much of a need to explicitly point this out as on Linux I think it is generally expected that you'd either use all the dxvk DLLs in your Wine prefix or none of them. Or at least I expect not doing so is rather niche.

We don't currently point out any of the other interdependencied like d3d10core needing d3d11 and dxgi or d3d11 needing dxgi.

Reply:
Or at least if this was added I think it might be better with a separate note paragraph thatbaldo mentions the other ones

Reply:
I think this is a really important information, which should be written somewhere. In my opinion software should be documented to minimize users' questions and to help future development (just in case new developer jouned the team, new contirutor wanted to help, the project has been forked etc). Even if for now some information seems to be obvious and you think most of the users won't try to use the software wrong way.

Reply:
Yes I agree it would make sense to mention it somewhere. But it should be done for all the DLLs. Not sure if it is the best location though.

I'll let the devs comment on it

Reply:
```
But it should be done for all the DLLs.
```
I agree
```
I'll let the devs comment on it
```
Thank you